core/src/main/resources/error/error-classes.json

```diff
@@ -70,6 +70,16 @@
       "Another instance of this query was just started by a concurrent session."
     ]
   },
+  "CONVERSION_INVALID_FORMAT" : {
```

Comment:
Can't we use the error class `INVALID_PARAMETER_VALUE` instead of it?

Reply:
Good point, fixed.